common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationRequest.java

```diff
@@ -96,6 +96,12 @@ public AuthorizationRequest(final String responseType,
         mScope = scope;
     }
 
+    /**
+     * Default constructor of AuthorizationRequest.
+     */
+    public AuthorizationRequest() {
```

Comment:
there's a default ctor without any args, so this ctor is unnecessary if it's empty.